CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- `opentelemetry-instrumentation-requests` Detect synthetic sources on requests.
```

Comment:
Seems like the functionality is being added for a couple of instrumentations? Can you update this changelog as well as the pr description.